var/spack/repos/builtin/packages/gaudi/package.py

```diff
@@ -167,8 +167,7 @@ def setup_run_environment(self, env):
         # environment as in Gaudi.xenv
         env.prepend_path("PATH", self.prefix.scripts)
         env.prepend_path("PYTHONPATH", self.prefix.python)
-        for d in self.libs.directories:
-            env.prepend_path("LD_LIBRARY_PATH", d)
+        # Note: ROOT dependency automatically sets up ROOT environment vars
```

Comment:
Gaudi also searches in `LD_LIBRARY_PATH` so this is also probably not going to going to work

Reply:
If this is about Gaudi/dd4hep needing additional search paths, shouldn't we also do as ROOT and add a `setup_dependent_X_environment`? (I assume `run` environment?)